src/server.php

```diff
@@ -213,6 +233,8 @@ public function search($param) {
         }
         $this->user_param = $param;
         $this->agency = $param->agency->_value;
+        // We know now the agency, so set it in the header.
+        $this->add_dbc_sla_header_agency($this->agency);
```

Comment:
Normalt kalder vi lokale funktioner med self:: istedet for $this, for at gøre det meget synligt at det er lokale private funktioner der kaldes.